ui/packages/mui-extends/src/Checkbox/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 Chaos Mesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { FormControl, FormControlLabel, FormHelperText, FormLabel } from '@mui/material'
+import { default as MuiCheckbox, CheckboxProps as MuiCheckboxProps } from '@mui/material/Checkbox'
+
+export type CheckboxProps = MuiCheckboxProps & {
+  /**
+   * The name of ths input field when using it in a form.
+   */
+  name?: string
```

Comment:
`MuiCheckboxProps` has already defined a `name` field. So this part is redundant.

Reply:
Updated! PTAL